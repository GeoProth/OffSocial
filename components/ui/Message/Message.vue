<template>
  <div
    v-if="type === 'error'"
    class="error-msg"
  >
    <h3 v-if="title">
      {{ title }}
    </h3>
    <p>{{ message }}</p>
  </div>
</template>
<script>
export default {
  name: "Message",
  props: {
    type: String,
    title: String,
    message: String
  }
};
</script>
<style lang="stylus" scoped>
.error-msg {
  width: 200px
  margin: auto
  border: 1px solid red
  border-radius: 4px
  background-color: lightpink
  font-size: 15px
  color: crimson
  text-align: center
  p {
    margin: 0
    padding: 0.5rem 0.5rem
  }
}
</style>
