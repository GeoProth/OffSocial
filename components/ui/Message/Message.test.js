import Message from "./Message";
describe("Message", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(Message);
  });
  test("renders", () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
