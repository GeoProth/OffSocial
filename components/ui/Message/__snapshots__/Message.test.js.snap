// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message renders 1`] = `undefined`;
